Classes/Private/Manager/Audio/OCTAudioEngine.m

```diff
@@ -0,0 +1,194 @@
+//
+//  OCTAudioEngine.m
+//  objcTox
+//
+//  Created by Chuong Vu on 5/24/15.
+//  Copyright (c) 2015 dvor. All rights reserved.
+//
+
+#import "OCTAudioEngine.h"
+@import AVFoundation;
+#define kInputBus 1
+#define kOutputBus 0
+
+@interface OCTAudioEngine ()
+{
+    AUGraph processingGraph;
+
+    AUNode ioNode;
+    AudioUnit ioUnit;
+    AudioComponentDescription ioUnitDescription;
+}
+
+@property (nonatomic, assign) double sampleRate;
```

Comment:
I don't think that you need this property. `sampleRate` can be obtained from shared AVAudioSession.